apps/prairielearn/src/components/GroupWorkInfoContainer.html.ts

```diff
@@ -0,0 +1,283 @@
+import { html } from '@prairielearn/html';
+
+import type { GroupConfig } from '../lib/db-types.js';
+import type { GroupInfo } from '../lib/groups.js';
+import { idsEqual } from '../lib/id.js';
+
+import { Modal } from './Modal.html.js';
+
+export function GroupWorkInfoCountainer({
+  groupConfig,
+  groupInfo,
+  userCanAssignRoles,
+  csrfToken,
+}: {
+  groupConfig: GroupConfig;
+  groupInfo: GroupInfo;
+  userCanAssignRoles: boolean;
+  csrfToken: string;
+}) {
+  return html`
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm bg-light py-4 px-4 border">
+          <h6 style="display:inline"><b>Group name: </b></h6>
+          <p style="display:inline" id="group-name">${groupInfo.groupName}</p>
+          <br />
+          ${groupConfig.student_authz_join
+            ? html`
+                <h6 style="display:inline"><b>Join code: </b></h6>
+                <p style="display:inline" id="join-code">${groupInfo.joinCode}</p>
+                <div class="mt-3">
+                  ${(groupConfig.minimum ?? 0) > 1
+                    ? html`
+                        This is a group assessment. Use your join code to invite others to join the
+                        group. A group must have
+                        ${groupConfig.minimum === groupConfig.maximum
+                          ? groupConfig.minimum
+                          : groupConfig.maximum
+                            ? `between ${groupConfig.minimum} and ${groupConfig.maximum}`
+                            : `at least ${groupConfig.minimum}`}
+                        students.
+                      `
+                    : html`
+                        This assessment can be done individually or in groups. Use your join code if
+                        you wish to invite others to join the group.
+                        ${groupConfig.maximum
+                          ? `A group must have no more than ${groupConfig.maximum} students.`
+                          : ''}
+                      `}
+                </div>
+              `
+            : ''}
+        </div>
+        <div class="col-sm bg-light py-4 px-4 border">
+          ${groupConfig.student_authz_leave
+            ? html`
+                <div style="display: flex; justify-content: flex-end">
```

Comment:
Nit: I imagine this was just copy/paste, but can we use Bootstrap classes (`d-flex`, etc.) instead?

Reply:
Done in 01ad7596b.